files/build_templates/dhcp_relay.service.j2

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=DHCP relay container
-Requires=updategraph.service
+Requires=updategraph.service database.service
```

Comment:
Is it necessary to explicitly add database.service here? This requires updategraph.service, which in turn requires config-setup.service which requires database.service. Did you find this chain of dependencies was not sufficient?

Reply:
I will look into this.

Reply:
I checked the service files and the current chain of dependencies is already sufficient.

Reply:
Did you test that restarting the database container actually causes the entire chain of dependent services to restart?

Reply:
Actually I tested it today but I want to test it again.

Reply:
Checked again and it did work.